samples/cellular/nrf_provisioning/README.rst

```diff
@@ -0,0 +1,137 @@
+.. _nrf_provisioning_sample:
+
+cellular: nRF Device provisioning
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use the :ref:`lib_nrf_provisioning` service on your device.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+The sample requires that private device key is installed on the device and the associated Device UUID is obtained from the Identity Service.
+
+.. note::
+   This sample requires modem firmware v2.0.0 or later.
+
+Overview
+********
+
+The sample shows how the device performs the following actions:
+
+* Connects to nRF Device provisioning Service.
+* Fetches available device specific provisioning commands.
+* Decodes the commands.
+* Acts on any AT commands, if available.
+* Reports the results back to the server.
+  In case of errors, stops processing the commands at the first error and reports it back to server.
+* Sends ``FINISHED`` response if all the previous commands are executed without errors provided and ``FINISHED`` is one of the set provisioning commands.
```

Comment:
I'm not sure what  'without errors provided and ``FINISHED`` is one of the set provisioning commands' means. Possible values for the command status in the Provisioning Service are PENDING, FAILED, IN_PROGRESS, SUCCEEDED, and SKIPPED. Does this refer to the command status or the status of something in the sample, which has different values?

Reply:
FINISHED refers to the provisioning command which tells to the device that provisioning process has been finished.